app/assets/javascripts/statuses.js

```diff
@@ -0,0 +1,84 @@
+$(function() {
+  $(".proxy-version").each(function(index, item) {
+    var item = $(item);
+    var url = item.data('url');
+    $.ajax({
+      type: 'get',
+      url:  url,
+      success: function(response) {
+        setItemMessage(item, response);
+      },
+      error: function(response) {
+        setItemErrorMessage(item, response)
+      }
+    })
+  });
+
+  $(".plugin-version").each(function(index, item) {
+    var item = $(item);
```

Comment:
atm there are multiple queries to the proxy at the same time, does it make sense to wait until the initial ping query is done successfully and only then do follow up queries?